### PR TITLE
Tsk 991

### PR DESCRIPTION
**Pull request by BerndBreier**

### CODE REVIEW

File: lib/taskana-core/src/main/java/pro/taskana/report/api/row/TimestampRow.java [hunk 1]
```diff
@@ -1,11 +1,12 @@
-package pro.taskana.impl.report.row;
```
Comment (mustaphazorgati) [inline lib/taskana-core/src/main/java/pro/taskana/report/api/row/TimestampRow.java]: this should be internal

File: lib/taskana-core/src/test/java/pro/taskana/ArchitectureTest.java [hunk 2]
```diff
@@ -16,17 +19,34 @@
...
 class ArchitectureTest {
-
   private static JavaClasses importedClasses;
 
+  DescribedPredicate<JavaClass> doNotContain(String... namesToBeExcluded) {
+    return new DescribedPredicate<JavaClass>("should not be checked") {
+      @Override
+      public boolean apply(JavaClass input) {
+        String matchingClassName =
+            Arrays.asList(namesToBeExcluded).stream()
```
Comment (mustaphazorgati) [inline lib/taskana-core/src/test/java/pro/taskana/ArchitectureTest.java]: Stream.of() :)

File: lib/taskana-core/src/test/java/pro/taskana/ArchitectureTest.java [hunk 3]
```diff
@@ -16,17 +19,34 @@
...
+                .filter(name -> input.getName().contains(name))
```
Comment (mustaphazorgati) [inline lib/taskana-core/src/test/java/pro/taskana/ArchitectureTest.java]: .filter(input.getName()::contains)

File: lib/taskana-core/src/main/java/pro/taskana/report/api/row/TimestampRow.java (hunk 1 above)
Comment (BerndBreier) [reply lib/taskana-core/src/main/java/pro/taskana/report/api/row/TimestampRow.java]: TimestampRow is returned by TimestampReport.getRow(). Therefore, I would consider it part of the API - after all it is a datastructure that the client code must understand ...

File: lib/taskana-core/src/test/java/pro/taskana/ArchitectureTest.java (hunk 2 above)
Comment (BerndBreier) [reply lib/taskana-core/src/test/java/pro/taskana/ArchitectureTest.java]: Granted, that's better

**Pull request merged**